src/common/workspace-store.ts

```diff
@@ -138,6 +138,7 @@ export class WorkspaceStore extends BaseStore<WorkspaceStoreModel> {
     super({
       configName: "lens-workspace-store",
     });
+    this.workspaces.get(WorkspaceStore.defaultId).enabled = true;
```

Comment:
This should be done in the initializer and `Workspace` constructor should take `enabled` as a field.